helpers/deploymentConfig.ts

```diff
@@ -57,18 +59,38 @@ export const xvsBridgeMethods = [
   "updateSendAndCallEnabled(bool)",
 ];
 
-export const NormalTimelockPermissions = [
-  "setTrustedRemoteAddress(uint16,bytes)",
-  "transferBridgeOwnership(address)",
-  "migrateMinterTokens(address,address)",
+export const xvsBridgeMethodsDest = [
+  "setSendVersion(uint16)",
+  "setReceiveVersion(uint16)",
+  "forceResumeReceive(uint16,bytes)",
+  "setOracle(address)",
+  "setMaxSingleTransactionLimit(uint16,uint256)",
+  "setMaxDailyLimit(uint16,uint256)",
+  "setMaxSingleReceiveTransactionLimit(uint16,uint256)",
+  "setMaxDailyReceiveLimit(uint16,uint256)",
+  "pause()",
+  "unpause()",
+  "removeTrustedRemote(uint16)",
+  "dropFailedMessage(uint16,bytes)",
+  "setPrecrime(address)",
+  "setMinDstGas(uint16,uint16,uint256)",
+  "setPayloadSizeLimit(uint16,uint256)",
+  "setWhitelist(address,bool)",
+  "setConfig(uint16,uint16,uint256,bytes)",
+  "sweepToken(address,address,uint256)",
+  "updateSendAndCallEnabled(bool)",
 ];
 
+export const XVSBridgeAdminMethods = ["setTrustedRemoteAddress(uint16,bytes)", "transferBridgeOwnership(address)"];
+
+export const XVSTokenDestMethods = ["migrateMinterTokens(address,address)", "setMintCap(address,uint256)"];
```

Comment:
I think we should add these entries:
* updateBlacklist(address,bool)
* pause()
* unpause()